src/target/stream.c

```diff
@@ -85,6 +118,25 @@ stumpless_open_stream_target( const char *name, FILE *stream ) {
   return NULL;
 }
 
+void
+stumpless_set_severity_color( struct stumpless_target *target, enum stumpless_severity severity, const char *escape_code )
+{
+  if (severity > 7)
+    raise_index_out_of_bounds("Severity value must be part of 'stumpless_severity' enum (0-7)", severity);
+
+  if (target->type != STUMPLESS_STREAM_TARGET)
+    raise_target_unsupported("This function is only supported for stream targets");
```

Comment:
This error message should be a localized string. The documentation for [localization](https://github.com/goatshriek/stumpless/blob/latest/docs/localization.md#adding-new-strings) describes how to add new localized strings: you just need to run `scripts/add_l10n_string.rb` with your new string.

Reply:
Thank you for all the feedback! I have been unable to add the localised string due to what may be a bug in the script (output shown below). I am not very fluent in ruby so I decided to just leave it as is for somebody else to work on.

Output:
`$ ruby scripts/add_l10n_string.rb SET_SEVERITY_COLOR_UNSUPPORTED_TARGET_ERROR_MESSAGE This function is only supported for stream targets
scripts/add_l10n_string.rb:59:in `each': Is a directory @ io_fillbuf - fd:7 <home-path>/stumpless/include/private/config/locale/. (Errno::EISDIR)
        from scripts/add_l10n_string.rb:59:in `block in <main>'
        from scripts/add_l10n_string.rb:53:in `each'
        from scripts/add_l10n_string.rb:53:in `<main>'`

Reply:
Ah, that is unfortunate. Thank you very much for sharing the details though, they did allow me to identify the issue and a fix. You can leave this out and I will go back and correct it afterwards - line 53 of the script needs to be changed to `each_child`. I'll submit a separate patch later.